themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig

```diff
@@ -112,6 +112,11 @@
 
           {{ content|without('localgov_news_date', 'localgov_news_categories', 'localgov_news_related', 'sign_in') }}
 
+          {% if node.localgov_news_related.value %}
```

Comment:
Remove these lines. Related content isn't displayed for anonymous users.

Reply:
Done